### PR TITLE
Reduced most of the noticeable movement jitter

### PR DESCRIPTION
**Pull request by Kasterisk**
Added `elapsed` parameter to `MicrobeCameraSystem::Run`
This is quite hackish and might need some more testing.
Closes: #911

### CODE REVIEW

**Comment by hhyyrylainen** [comment]
add to whitelist

File: src/microbe_stage/microbe_camera_system.cpp [hunk 1]
```diff
@@ -56,8 +56,8 @@ void
 
         if(cameraPos.Members._Position != targetPos) {
 
-            cameraPos.Members._Position =
-                cameraPos.Members._Position.Lerp(targetPos, CAMERA_FLOW);
+            cameraPos.Members._Position = cameraPos.Members._Position.Lerp(
+                targetPos, elapsed / CAMERA_FLOW);
```
Comment (hhyyrylainen) [inline src/microbe_stage/microbe_camera_system.cpp]: Does it make sense to have it this way? Why not increase the CAMERA_FLOW constant and do it like `CAMERA_FLOW * elapsed` that way my brain could understand what this is doing much easier?

Comment (Kasterisk) [reply src/microbe_stage/microbe_camera_system.cpp]: Yes it's more sensible that way, but the CAMERA_FLOW value has to be like 7 or 8.

Comment (hhyyrylainen) [reply src/microbe_stage/microbe_camera_system.cpp]: I don't see a problem with setting CAMERA_FLOW to be 7. There could also be another constant multiplied together with them to keep the CAMERA_FLOW small.

Comment (Kasterisk) [reply src/microbe_stage/microbe_camera_system.cpp]: Ok, I need to test the values for a sec

**Comment by hhyyrylainen** [comment]
Could someone maybe record a video comparing how this looks?

**Comment by Kasterisk** [comment]
Sure, here is the video https://youtu.be/zAzyTBOzgtU

**Pull request merged**